docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -0,0 +1,281 @@
+
+    <para>There are several ways to customize a platform deployment. We
+    recommend using methods that allow you to best take advantage of the
+    configuration as code (CaC) practices. Configuration as code is the
+    standard of managing configuration files in a version control system or
+    repository.</para>
+
+    <para>The following is a list of common customization techniques:</para>
+
+    <itemizedlist>
+      <listitem>
+        <para>The first way to override a setting in the default configuration
+        on the command line using the <emphasis role="bold">--set</emphasis>
+        parameter. </para>
+
+        <para>This is the easiest, but the least compliant with CaC
+        guidelines. It is also harder to keep track of overrides this
+        way.</para>
+      </listitem>
+
+      <listitem>
+        <para>The second way is to download and modify the default YAML file
+        (..\helm\hpcc\values.yaml). This could comply with CaC guidelines if
+        you place that file under version control, but it makes it harder to
+        utilize a newer default configuration when one becomes
+        available.</para>
+      </listitem>
+
+      <listitem>
+        <para>The third way, is the one we typically use. Use the default
+        configuration plus a customization YAML file and use the -f parameter
+        to the helm command to use it. This uses the default configuration and
+        only overrides the settings specified in the customization
+        YAML..</para>
+
+        <para>For this tutorial, we will use the third method to stand up a
+        platform with all the default settings but add some customizations. In
+        the first example, instead of one Roxie, it will have two. In the
+        second example, it will add a second 10-way Thor.</para>
+      </listitem>
+    </itemizedlist>
+
+    <sect2 id="CustTutorial1" role="nobrk">
+      <title>Create a Custom Configuration Chart for Two Roxies</title>
+
+      <orderedlist>
+        <listitem>
+          <para>Create a new text file and name it <emphasis
+          role="bold">TwoRoxies.yaml</emphasis> and open it in a text
+          editor.</para>
+
+          <para>You can use any text editor.</para>
+        </listitem>
+
+        <listitem>
+          <para>Prepare to copy portions of default YAML file
+          (..\helm\hpcc\values.yaml) </para>
+
+          <para>You can use the HPCC-Platform repository or the helm-chart
+          repository. </para>
+
+          <para>https://github.com/hpcc-systems/HPCC-Platform </para>
+
+          <para>https://github.com/hpcc-systems/helm-chart </para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire <emphasis role="bold">roxie:</emphasis>
+          section and paste it into the new TwoRoxies.yaml file.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire contents, except the first line, and paste it
```

Comment:
personal: I would add (roxie:) to this sentence, as : "except for the first line (roxie:), and paste it"